gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -1292,6 +1331,33 @@ MM_ParallelGlobalGC::fixHeapForWalk(MM_EnvironmentBase *env, UDATA walkFlags, ui
 	return fixedObjectCount;
 }
 
+/**
+ * Clearing all dead multi-slot objects, whether linked or unlinked.
+ * Currently only called at snapshot time.
+ * @param[in] env The environment for the calling thread.
+ * @param[in] walkFunction the callback function to operate on each object.
```

Comment:
in the function description, params and return don't end with a period
```suggestion
 * @param[in] env The environment for the calling thread
 * @param[in] walkFunction the callback function to operate on each object
```

Reply:
Should be fixed now